invenio_rdm_migrator/actions/base.py

```diff
@@ -67,18 +44,23 @@ class LoadAction(Action, ABC):
     This generates the corresponding SQL operations to perform.
     """
 
-    def __init__(self, tx_id, pks=None):
+    data_cls: ClassVar[type[LoadData]] = None
```

Comment:
we use typing.... 😏  This would help a lot with some "type" assertions needed in the load steps 🚀

Reply:
The assertions are still helpful, since the typing is just a "suggestion" to the editor. But yeah, I felt that without them it would be difficult to clearly define what is expected (classes vs. instances).